src/dag/read.ts

```diff
@@ -30,9 +33,14 @@ export class Read {
     } else {
       refs = [];
     }
-    return Chunk.read(hash, data, refs);
+    return readChunk(hash, data, refs);
   }
 
+  createChunk = <V extends ReadonlyJSONValue>(
```

Comment:
I would expect this to be on dag.Write.  Why would you need to create a chunk on a read?

Reply:
Good question. Will look into it